frontend/src/components/profile/components/documents/documents.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { IDocument } from 'common/interfaces';
+import { Modal, Button, Radio } from 'components/common';
+import { ButtonColor, ButtonStyleType, DocumentStatus, ButtonType } from 'common/enums';
+import { InputChangeEvent } from 'common/types';
+import { ProfileActionCreator } from 'store/slices';
+import { useDispatch } from 'react-redux';
+import clsx from 'clsx';
+import styles from './styles.module.scss';
+
+type Props = {
+  document: IDocument
+};
+
+const Status = {
+  [DocumentStatus.VERIFIED]: 'verified',
+  [DocumentStatus.IN_REVIEW]: 'in review',
+};
+
+const Documents: React.FC<Props> = ({ document }) => {
+  const [isModalOpen, setIsModalOpen] = React.useState<boolean>(false);
+  const [radioValue, setRadioValue] = React.useState<DocumentStatus>(document.status);
+  const dispatch = useDispatch();
+  const handleChangeRadio = (event:InputChangeEvent) => {
+    setRadioValue(event.target.value as DocumentStatus);
+  };
+  const handleSubmit = () => {
+    const update:IDocument = { ...document, status: radioValue };
+    dispatch(ProfileActionCreator.editUserDocument(update));
+    setIsModalOpen(false);
+  };
+  const handleModalOpen = () => {
+    setIsModalOpen(!isModalOpen);
+  };
+  return (
+    <div className={styles.documentContainer}>
+      <span>Document status: </span>
+      <span className={clsx(styles[document.status], styles.label)}>
+        {Status[document.status]}
+      </span>
+      <div className={styles.checkButton}>
+        <Button
+          label="Check document"
+          color={ButtonColor.GRAY_LIGHT}
+          styleType={ButtonStyleType.WITHOUT_BORDER}
+          hasHiddenLabel={false}
+          onClick={handleModalOpen}
+        />
+      </div>
+      <Modal isShow={isModalOpen}>
+        <div className={styles.formContainer}>
+          <button className={styles.closeButton} onClick={handleModalOpen} type="button">
+              &#10060;
+          </button>
+          <form className={styles.form} onSubmit={handleSubmit}>
+            <div className={styles.radio}>
+              <Radio
+                options={[
+                  {
+                    label: Status[DocumentStatus.VERIFIED],
+                    value: DocumentStatus.VERIFIED,
+                  },
+                  {
+                    label: Status[DocumentStatus.IN_REVIEW],
+                    value: DocumentStatus.IN_REVIEW,
+                  },
+                ]}
```

Comment:
```suggestion
                options={verifyDocumentOptions}
```
Create options outside of the component